plugins/harness-iacm/src/components/ExampleFetchComponent/ExampleFetchComponent.tsx

```diff
@@ -0,0 +1,308 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Table,
+  TableColumn,
+  Progress,
+  ResponseErrorPanel,
+} from '@backstage/core-components';
+import useAsync from 'react-use/lib/useAsync';
+
+export const exampleUsers = {
```

Comment:
Do we require so many example users??

Reply:
i dont think we really need this component, I will add a proper test for the components I have added